tensorflow/contrib/hooks/BUILD

```diff
@@ -0,0 +1,52 @@
+# Description:
+#   Contains `SessionRunHook`s for use with `MonitoredSession` and the
+#   wrappers around it.
+
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["LICENSE"])
+
+package(default_visibility = ["//tensorflow:__subpackages__"])
+
+py_library(
+    name = "hooks",
+    srcs = [
+        "__init__.py",
+        "python/training/__init__.py",
+        "python/training/profiler_hook.py",
+    ],
+    srcs_version = "PY2AND3",
+    deps = [
+        "//tensorflow/contrib/framework:framework_py",
+        "//tensorflow/python:framework",
+        "//tensorflow/python:framework_for_generated_wrappers",
+        "//tensorflow/python:state_ops",
+        "//tensorflow/python:training",
+        "//tensorflow/python:variables",
+    ],
+)
+
+py_test(
+    name = "profiler_hook_test",
+    size = "small",
+    srcs = ["python/training/profiler_hook_test.py"],
+    srcs_version = "PY2AND3",
+    deps = [
+        ":hooks",
+        "//tensorflow/python:client_testlib",
+        "//tensorflow/python:framework_for_generated_wrappers",
+        "//tensorflow/python:framework_test_lib",
+        "//tensorflow/python:platform_test",
+    ],
+)
+
+filegroup(
+    name = "all_files",
```

Comment:
This rule also needs to be referenced from `all_opensource_files` rule in `tensorflow/BUILD` file.
Otherwise it will be excluded from our pip packages.